### PR TITLE
EDU-1018 Set up a license list and selector

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-1018

### CODE REVIEW

File: src/bootstrap/server-config.js [hunk 1]
```diff
@@ -50,6 +51,7 @@ const configSchema = joi.object({
   }).allow(null),
   basicAuthUsers: joi.object().default({}),
   plugins: joi.array().items(joi.string().required()).default(['markdown', 'image']),
+  allowedLicenses: joi.array().items(joi.string().valid(...Object.keys(spdxLicenseList))).default(Object.keys(spdxLicenseList)),
```
Comment (adrianaluchian) [inline src/bootstrap/server-config.js:54]: Let's add this to the `Readme`, maybe including a small example, like `'CC0-1.0', 'CC-BY-4.0']`?

**Comment by adrianaluchian** [comment]
Looks very good and clean!

File: src/bootstrap/server-config.js (hunk 1 above)
Comment (ahelmberger) [reply src/bootstrap/server-config.js:54]: Done! Thanks for reminding me each time :blush:

Comment (adrianaluchian) [reply src/bootstrap/server-config.js:54]: 😄 Sure thing!

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**